Library/Homebrew/requirement.rb

```diff
@@ -83,12 +83,10 @@ def modify_build_environment
     # PATH.
     # This is undocumented magic and it should be removed, but we need to add
     # a way to declare path-based requirements that work with superenv first.
-    if @satisfied_result.is_a?(Pathname)
-      parent = @satisfied_result.parent
-      unless ENV["PATH"].split(File::PATH_SEPARATOR).include?(parent.to_s)
-        ENV.append_path("PATH", parent)
-      end
-    end
+    return unless @satisfied_result.is_a?(Pathname)
+    parent = @satisfied_result.parent
+
+    ENV.append_path("PATH", parent) unless ENV["PATH"].split(File::PATH_SEPARATOR).include?(parent.to_s)
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.